extensions/blocks/instagram-gallery/editor.scss

```diff
@@ -1,3 +1,22 @@
+.wp-block-jetpack-instagram-gallery__gallery-loading {
+	position: absolute;
+	right: 24px;
+}
+.wp-block-jetpack-instagram-gallery__count-notice {
+	.components-notice__content {
+		margin: 0;
+		line-height: inherit;
+		padding-right: 0;
+	}
+	.components-notice.is-dismissible {
+		padding-right: 12px;
```

Comment:
As mentioned in https://github.com/Automattic/jetpack/pull/15329/files#r406117663, I'd probably remove the horizontal margins here, and let the notice fill the whole sidebar width.

Reply:
done